components/post_view/reaction_list/reaction_list.jsx

```diff
@@ -5,8 +5,13 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import {postListScrollChange} from 'actions/global_actions.jsx';
+import {emitEmojiPosted} from 'actions/post_actions.jsx';
 
 import Reaction from 'components/post_view/reaction';
+import EmojiPickerOverlay from 'components/emoji_picker/emoji_picker_overlay.jsx';
+
+const DEFAULT_EMOJI_PICKER_RIGHT_OFFSET = 15;
+const EMOJI_PICKER_WIDTH = 260;
```

Comment:
Not sure why `260`? I'm seeing different width (`278px`) in this line: https://github.com/mattermost/mattermost-webapp/blob/master/sass/components/_emoticons.scss#L58

Reply:
Oh I just set `260` when I was experimenting with the positioning, if it's properly defined as `278` then I'll set it to that instead.

Reply:
Thanks, then better to change to `278` and let's see what will happen.

Reply:
The reason I set it to `260` originally was because that way the corner of the emoji picker would line up exactly with the corner of the `+` button. I'll change it to `278` and you guys can play around with it and see if the positioning is ok.

Reply:
I see what you mean. So I guess it would just be better to just change the name to like: 
```
const EMOJI_PICKER_WIDTH_OFFSET = 260;
```

Reply:
No worries, I'll change it back and rename it to what you suggested.